types/google-apps-script/google-apps-script.calendar.d.ts

```diff
@@ -1,10 +1,10 @@
-// Type definitions for Google Apps Script 2015-11-12
+// Type definitions for Google Apps Script 2017-05-12
 // Project: https://developers.google.com/apps-script/
 // Definitions by: motemen <https://github.com/motemen/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
-
+/// <reference path="google-apps-script.types.d.ts" />
```

Comment:
Thanks, I didn't know [`<reference types=...>` directive](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-) and have just read the doc.

Are you saying this line should be `/// <reference types="google-apps-script" />`? I think it causes kind of recursive reference and does not achieve what I wanted to do; I just wanted to import type declarations such as `GoogleAppsScript.Integer` which is declared in google-apps-script.types.d.ts.

Maybe whole these references could be deleted, as index.d.ts references all files.